src/py/flwr/simulation/ray_transport/ray_actor.py

```diff
@@ -25,18 +25,12 @@
 from ray import ObjectRef
 from ray.util.actor_pool import ActorPool
 
-from flwr import common
-from flwr.client import Client, ClientFn
+from flwr.client.clientapp import ClientApp
 from flwr.common.context import Context
 from flwr.common.logger import log
-from flwr.simulation.ray_transport.utils import check_clientfn_returns_client
+from flwr.common.message import Message
 
-# All possible returns by a client
-ClientRes = Union[
-    common.GetPropertiesRes, common.GetParametersRes, common.FitRes, common.EvaluateRes
-]
-# A function to be executed by a client to obtain some results
-JobFn = Callable[[Client], ClientRes]
+FlowerFn = Callable[[], ClientApp]
```

Comment:
How about `ClientAppFn` or `ClientAppFactory`? After refactoring some of the other names that had `Flower` in them, I prefer not to have `Flower` in names anymore since those tend to be difficult to refactor (e.g., searching for `Flower` yields every single file in the project since it's mentioned in the copyright notice).